### DISCUSSION TITLE
[SOLVED] Is ALVR only supported along with Steam?

### DISCUSSION THREAD

**Discussion by Hi-Angel** [alvr-org/ALVR#2530] [General]
I stumbled upon a bug, where I'm trying to launch a non-steam FOSS game *(The Dark Mod VR)*, and when I do that ALVR does weird things… On the first launch ALVR ran Steam for whatever reason *(it didn't in later launches)*. But more importantly, every time I run game with ALVR the game just crashes. It's definitely a ALVR bug, because running the game with WiVRn works.

But before reporting a bug, I noticed that README.md mentiones "Steam" in "Requirements" field. So I'm not sure which bug to report:

1. Incorrect project description *(because it positions itself as a general streaming app and not Steam)*
2. The one with the crash

**Comment by TayKara** [comment]
My I ask what happens if you add the game as a non-steam game in your Steam library, and start it from there ?

**Comment by zmerp** [comment]
ALVR is implemented as a SteamVR driver plugin. Monado is not supported currently but there is some effort in adding support to it. No ETA but you could expect some working prototype by the end of march next year.

**Reply by Hi-Angel** [reply]
I see, thank you!
